sys/arm64/include/elf.h

```diff
@@ -72,6 +93,8 @@ __ElfType(Auxinfo);
 
 #define	ELF_MACHINE_OK(x) ((x) == (ELF_ARCH))
 
+#define	ELF_IS_CHERI(hdr) (0)
```

Comment:
Can you not implement this correctly now rather than later?

Reply:
No, I already have the fix for this, it's just part of a change to add CheriABI support. I'm trying to get these out of my development tree.

Reply:
Sure, so let's define the flag and check it here, even if it doesn't work? I want to avoid touching the same line multiple times wherever possible; there's no point committing something that is known to need changing.